cmd/geth/chaincmd.go

```diff
@@ -219,6 +221,34 @@ Use "ethereum dump 0" to dump the genesis block.`,
 		},
 		Category: "BLOCKCHAIN COMMANDS",
 	}
+	statDbCommand = cli.Command{
```

Comment:
nitpicks, would be nice to have a new file `dbcmd.go` and put all db related commands there(also make the new category `DATABASE COMMANDS`.